### PR TITLE
fix(generator): cmf example big update

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

The CMF app doesn't work. #28 
It just display an Hello string

**What is the new behavior?**

THe CMF app is working and install all the stack for you (containers, theme, icon, ...)
It displays a welcome page.

### CODE REVIEW

File: packages/containers/src/index.js [hunk 1]
```diff
@@ -1,45 +1,208 @@
...
 	IconsProvider,
 	Layout,
+	// List,
+	// Notification,
+	Progress,
+	// SidePanel,
+	Status,
 	TooltipTrigger,
 	Typeahead,
 } from 'react-talend-components';
```
Comment (sdwvit) [inline packages/containers/src/index.js]: https://gist.github.com/caridy/839d5359321604a12648#file-export-syntax-js-L29

please check a way to reduce # of lines on gist, however it's stage1 so idk if we can use it.

File: packages/generator/generators/react-cmf/templates/src/app/configure.js [hunk 2]
```diff
@@ -10,10 +10,4 @@ export default {
 	initialize() {
 		registerComponent('App', App);
 	},
-	getStore() {
```
Comment (sdwvit) [inline packages/generator/generators/react-cmf/templates/src/app/configure.js]: Does it mean I cannot access store for dispatching outside of components with connect()?

**Comment by sdwvit** [comment]
please fix tests

**Comment by jmfrancois** [comment]
I will do that tomorrow

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://54.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://54.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://54.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://54.talend.surge.sh)

**Pull request merged**